spec/controllers/passwords_controller_spec.rb

```diff
@@ -3,76 +3,80 @@
 require 'spec_helper'
 
 describe Api::V1::PasswordsController do
+  let!(:user) { create(:user, password: 'mypass123') }
+  let(:password_token) { user.send(:set_reset_password_token) }
```

Comment:
check the style guidelines, but it will look nicer if you align the params there